src/lib/rule-finder.js

```diff
@@ -75,6 +75,11 @@ function RuleFinder(specifiedFile) {
     return getSortedRules(currentRules)
   }
 
+  // get all the current rules' particular configuration
+  this.getCurrentRulesDetailed = function getCurrentRulesDetailed() {
+    return config.rules
+  }
+
```

Comment:
Strange!! [why is this still in diff](https://github.com/sarbbottam/eslint-find-rules/blob/master/src/lib/rule-finder.js#L78-L81)?

Do you need to rebase this with `upstream/master`?